source/direct_debit/index.html.md.erb

```diff
@@ -255,96 +133,197 @@ Required arguments:
+
+`GET
+/v1/directdebit/mandates?from_date=2018-10-01T13:30:00Z&to_date=2018-10-14T15:00:00Z&display-size=500&page=2`
+
+This would return payments 501 to 1000.
+
+## Mandate statuses
 
-<br>
 <div style = "overflow-x:auto;">
   <table style = "width:100%">
     <tr>
     <th nowrap>State</th>
     <th>Description</th>
     </tr>
     <tr>
-    <td nowrap><code>CREATED</code></td>
-    <td>Your service made an API call to create the agreement.</td>
+    <td nowrap><code>created</code></td>
+    <td>Your service made an API call to create the mandate.</td>
     </tr>
-    <td nowrap><code>STARTED</code></td>
-    <td>Your service sent an API request to GOV.UK Pay.</td>
+    <td><code>started</code></td>
+    <td>The user has visited the <code>next_url</code> and landed on the GOV.UK Pay
+page to enter their account information.</td>
     </tr>
-    <td nowrap><code>PENDING</code></td>
+    <td><code>pending</code></td>
     <td>Indicates that account details have been submitted to the
 payment service provider (PSP), but the PSP is not yet able to confirm whether
-the payment is complete.</td>
+the mandate has been successfully set up.</td>
+    </tr>
+    <tr>
+    <td><code>active</code></td>
+    <td>The PSP has confirmed that the mandate has been succesfully set up.</td>
+    </tr>    
+    <tr>
+    <td><code>inactive</code></td>
+    <td>You cannot collect payments from the mandate, for example because the user cancelled it.
+    </tr>    
+    <tr>
+    <td><code>cancelled</code></td>
+    <td>When the user entered their bank account information, they answered <b>No</b> to <b>Are you allowed to authorise Direct Debits on this account?</b>.</td>
+    </tr>   
   </table>
 </div>
 
-The payment reaches one of several finished states.
+## Payment statuses
 
 <br>
 <div style = "overflow-x:auto;">
   <table style = "width:100%">
     <tr>
-    <th nowrap>Finished state</th>
+    <th nowrap>State</th>
     <th>Description</th>
+    </tr>
+    <td nowrap><code>pending</code></td>
+    <td>The payment has been requested, but the PSP is not yet able to confirm whether
+the payment is complete.</td>
     </tr>
     <tr>
-    <td><code>SUCCESS</code></td>
-    <td>Payment succeeded.</td>
+    <td><code>success</code></td>
+    <td>The payment succeeded.</td>
     </tr>
-    <td><code>FAILED</code></td>
-    <td>Payment failed.</td>
+    <td><code>failed</code></td>
+    <td>The payment failed.</td>
     </tr>
-    <td><code>CANCELLED</code></td>
+    <tr>
+    <td><code>cancelled</code></td>
     <td>The service cancelled the payment.</td>
     </tr>
-    <td><code>ERROR</code></td>
-    <td>Something went wrong with GOV.UK Pay or the underlying Payment Service Provider.</td>
-    </tr>
   </table>
 </div>
 
-### Differences compared to card payment statuses
-
-The states `created`, `started` and `submitted` are all represented as
-a single state: `STARTED`.
-
-Direct Debit payments can have the non-terminal state `PENDING`.
-
 ## Reference numbers
```

Comment:
@katiebates Can I double-check this section will still be correct as is? (It's not all shown here: [here's the link to the live section](https://docs.payments.service.gov.uk/direct_debit/#reference-numbers))

Reply:
sorry i missed this. all correct apart from reference numbers being shown in the GOV.UK Pay admin tool - that's the long term vision, but won't be available to begin with. plus we don't send those reference numbers to GC so they won't be available in their admin tool. so just available in the API for now.

Reply:
Ah good to know - thanks Katie. Will remove that bit.